### DISCUSSION TITLE
Celebrating 500,000 Users - Revolt

### DISCUSSION THREAD

**Discussion by giscus** [stoatchat/discussions#851] [Announcements]
# Celebrating 500,000 Users - Revolt

Revolt breaks the ceiling on five-hundred thousand registered users

https://revolt.chat/posts/500K-users

**Comment by Andre601** [comment]
Nice to see such a growth, tho it's also a bit bitter-sweet that said recent growth was due to some... events in specific countries revolving around another chat platform. But as long as Revolt can sustain itself...

(Also, really hope the issue with the UI of the mobile app being partially inside the phone's navigation, if it uses virtual buttons or what you call them, will be fixed soon, as it's so far the only real issue I have with the app...)

**Comment by randomysussy** [comment]
Nice job revolt! i made this account a few months ago just for revolt,  and now it went from 170,000 (estimate) -> 500,000 users  hope many more good stuff come to revolt!

**Comment by DeclanChidlow** [comment]
At 531K now!

**Comment by MagmaSlime123** [comment]
Congrats!

**Comment by bombasztikus** [comment]
gg

**Comment by xRevez** [comment]
Cool, wonder how long will take to hit 1M users. Good luck with the project!

**Reply by ghost** [reply]
hmm, lets math, so it took `4` years to get to `500k`, so if we `4 * 2` (because `500k * 2 = 1m`) then we get `8` years.

though this isn't accurate at all, since who knows if and when like `200k` people start using revolt

**Reply by Andre601** [reply]
I feel like this math doesn't quite add up. Especially given that 4/5ths of this milestone were reached within a year (Since their [100k milestone announcement](https://github.com/orgs/revoltchat/discussions/562)).

So it would be a bit more accurate to find out how much Revolt has increased its growth in this past year compared to the last 3 years up until the 100k Milestone (Basing it off your 4 year mention).

So it would effectively boil down to this formula here:
```math
100/(100000/3)*400000=1200
```

What we do here is get the average user growth for a year by dividing 100k by 3, which gives us $33333.33\overline{3}$ and multiply it by the total growth we had last year, which was 400k, giving us `1200%` of growth in just one year compared to the other three.

So, if this growth keeps like that could the 1M milestone be reached within less than 2 years (Roughly 1 year and 3 months).

(Disclaimer: I don't know if my math is accurate here. Feel free to correct me and explain where I was wrong. Be nice tho, thanks).

**Comment by omardotdev** [comment]
lessgo

**Comment by ghost** [comment]
Good job the team !
